bigquery/samples/list_datasets_projects.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Command-line skeleton application for BigQuery API.
+
+This is the sample for this page:
+
+    https://cloud.google.com/bigquery/docs/managing_jobs_datasets_projects
+
+In order to run it, you must be authenticated using one of the methods listed
+here:
+
+    https://developers.google.com/identity/protocols/application-default-credentials#howtheywork
+
+Usage:
+
+    $ python list_datasets_projects.py <project-id>
+
+where <project-id> is the id of the developers console [1] project you'd like
+to list the bigquery datasets and projects for.
+
+[1] https://console.developers.google.com
+"""
+
+import argparse
+import pprint
+
+from urllib2 import HTTPError
+
+from apiclient import discovery
+
+from oauth2client.client import GoogleCredentials
+
+
+# [START list_datasets]
+def list_datasets(service, project):
+    try:
+        datasets = service.datasets()
+        list_reply = datasets.list(projectId=project).execute()
+        print 'Dataset list:'
```

Comment:
use print as a function ``print()``, so that this will work without modification on py3.